hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -175,6 +175,13 @@ private Message process(final SCMRatisRequest request) throws Exception {
     }
   }
 
+  @Override
+  public void notifyLogFailed(Throwable ex,
+                              RaftProtos.LogEntryProto failedEntry) {
```

Comment:
Please take care of the indent here.

Reply:
> Please take care of the indent here.

@ChenSammi thx Sammi for the suggestion, I have updated it.